projects/plugins/crm/phpcs.xml.dist

```diff
@@ -14,7 +14,6 @@
 	<exclude-pattern>./vendor/*</exclude-pattern>
```

Comment:
FYI, I think this file is no longer needed. PHPCS is done at the monorepo level now.

Reply:
Yeah, this one doesn't seem to provide much value anymore. I'll remove it in a separate PR - thank you!